Docs/source/building/juwels.rst

```diff
@@ -0,0 +1,64 @@
+.. _building-juwels:
+
+Juwels (JSC)
+============
+
+The `Juwels supercomputer <https://www.fz-juelich.de/ias/jsc/EN/Expertise/Supercomputers/JUWELS/JUWELS_node.html>`_ is located at JSC.
+
+See `this page <https://apps.fz-juelich.de/jsc/hps/juwels/quickintro.html>`__ for a quick introduction.
+
+Installation
+------------
+
+Use the following commands to download the WarpX source code and switch to the correct branch:
+
+.. code-block:: bash
+
+   mkdir ~/src
+   cd ~/src
+
+   git clone https://github.com/ECP-WarpX/WarpX.git warpx
+   git clone --branch QED https://github.com/ECP-WarpX/picsar.git
+   git clone --branch development https://github.com/AMReX-Codes/amrex.git
+
+We use the following modules and environments on the system.
+
+.. code-block:: bash
+
+   # please set your project account
+   export proj=<yourProject>
+
+   # required dependencies
+   module load GCC
+   module load OpenMPI
+   module load CUDA
+
+   # AMReX specifics
```

Comment:
```suggestion
   # JEWELS' job scheduler may not map ranks to GPUs,
   # so we give a hint to AMReX about the node layout.
   # This is usually done in Make.<supercomputing center> files in AMReX
   # but there is no such file for JSC yet.
```

Reply:
Yep, just FYI: AMReX tries to remove most of these domain-specific files (some HPC centers might stay).